pkg/datasources/parameters.go

```diff
@@ -15,7 +16,7 @@ var parametersSchema = map[string]*schema.Schema{
 	"parameter_type": {
 		Type:         schema.TypeString,
 		Optional:     true,
-		Default:      "SESSION",
+		Default:      "ACCOUNT",
```

Comment:
Is this backward compatible?

Reply:
the old behavior "SESSION" was actually the same as "ACCOUNT", hence setting it as "ACCOUNT" is the most sensible. This is generally what people want to do anyways